src/Functions/DateTimeTransforms.h

```diff
@@ -1343,6 +1343,30 @@ struct ToYYYYMMDDhhmmssImpl
     using FactorTransform = ZeroTransform;
 };
 
+struct ToDateTimeComponentsImpl
```

Comment:
I just found out about #43466 and as far as I see, a similar fix (`DateLUTImpl::Time` instead of `time_t`, l. 1356) is needed.

Reply:
Thank you for noticing it. I will update it in my branch in about 3 hours from now.

Reply:
I replaced time_t with DateLUTImpl::Time and pushed my changes. It seems it's OK now with Docs Check?